FreeRTOS/FreeRTOS-Kernel
# Cortex-M35P: Add Cortex-M35P port

Cortex-M35P: Add Cortex-M35P port

Description
-----------
The Cortex-M35P support added to kernel. The port hasn't been validated yet with TF-M. Hence TF-M support is not included in this port.

Test Steps
-----------
The port is validated with a blinky example https://github.com/FreeRTOS/FreeRTOS-Partner-Supported-Demos/pull/9 

Checklist:
----------


- [ x ] I have tested my changes. No regression in existing tests.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
